Libplanet/Blocks/Block.cs

```diff
@@ -97,10 +96,10 @@ private Block(RawBlock rb)
         public DateTimeOffset Timestamp { get; }
 
         [IgnoreDuringEquals]
-        public IEnumerable<Transaction<T>> Transactions { get; }
+        public Transaction<T>[] Transactions { get; }
 
         [IgnoreDuringEquals]
-        public IEnumerable<Transaction<T>> OrderedTransactions { get; }
+        public Transaction<T>[] OrderedTransactions { get; }
```

Comment:
In the same way to the above.

```suggestion
        public ImmutableArray<Transaction<T>> OrderedTransactions { get; }
```